internal/server/iigo.go

```diff
@@ -21,12 +21,22 @@ func (s *SOMASServer) runIIGO() error {
 }
 
 func (s *SOMASServer) updateIIGOTurnHistory(clientID shared.ClientID, pairs []rules.VariableValuePair) {
-	s.gameState.IIGOHistory = append(s.gameState.IIGOHistory,
-		shared.Accountability{
-			ClientID: clientID,
-			Pairs:    pairs,
-		},
-	)
+	if _, ok := s.gameState.IIGOHistory[s.gameState.Turn]; ok {
+		s.gameState.IIGOHistory[s.gameState.Turn] = append(s.gameState.IIGOHistory[s.gameState.Turn],
+			shared.Accountability{
+				ClientID: clientID,
+				Pairs:    pairs,
+			},
+		)
+	} else {
+		s.gameState.IIGOHistory[s.gameState.Turn] = []shared.Accountability{
+			{
+				ClientID: clientID,
+				Pairs:    pairs,
+			},
+		}
+	}
```

Comment:
ah sorry, it isnt neded
![image](https://user-images.githubusercontent.com/33488131/103466716-36a51900-4d3f-11eb-8ddd-297bece05ce3.png)

This is good chacne to learn how to `rebase` or `drop` the commit

Reply:
Oh god
No

Reply:
Ok ok

Reply:
Let me try this

Reply:
Ok I think that worked